packages/atomic/src/components/common/generated-answer/styles/rephrase-buttons.pcss

```diff
@@ -9,10 +9,14 @@
   &:hover .rephrase-btn-label {
     @apply text-primary;
   }
+
+  &.active .rephrase-btn-label {
+    @apply text-white;
+  }
 }
 
 [part='rephrase-button'] {
-  --height: 2.2em;
+  --height: 21px;
```

Comment:
The previous height was relative to the font size just like the icon height. Did you try increasing the font size in the browser? Does it render properly?

Reply:
My point is that `1em` typically equals `16px`. It means an icon size of `1.1em` is almost `18px`. Things change when the end user decides to scale up the font size to 150% for instance. In this case `1em` becomes `24px` and `1.1em` becomes `26px`. At some point, an icon that is `1.1em` tall might not fit in a `21px` (fixed size) container. Usually we try and use relative dimensions everywhere so it all scale properly according to the font size.

Reply:
hmm I see, It renders properly when tested increasing the font size, let me know if there is something i didn' catch 🙏

Reply:
I tested by changing the `Font size` browser setting instead of `Page zoom`.

Reply:
@lbergeron  fixed also for citations as discussed :) Thanks for raising this point! 
Large:
<img width="1007" alt="Screenshot 2024-03-14 at 2 09 38 PM" src="https://github.com/coveo/ui-kit/assets/119955059/8f7e90ee-c0ea-4f45-aaee-f3697a8cfe14">

Small:
<img width="1030" alt="Screenshot 2024-03-14 at 2 06 54 PM" src="https://github.com/coveo/ui-kit/assets/119955059/78e01fd4-c0a0-41a1-bd0e-d9ab86222b11">